api/v1/installation_types.go

```diff
@@ -660,3 +664,13 @@ type CertificateManagement struct {
 	// +optional
 	SignatureAlgorithm string `json:"signatureAlgorithm,omitempty"`
 }
+
+// ConditionTypes describes Installation CR's status condition type.
+// One of: Ready, Degraded, Progressing.
+type ConditionType string
+
+const (
+	ConditionTypeReady       ConditionType = "Ready"
```

Comment:
Please add comments for each. You can copy them from tigerastatus_types.go.

Please note that each comment should start with the variable name. (This is currently not the case in tigerastatus_types.go and should be rectified).
Example:
```
// ComponentAvailable indicates that the component is healthy.
	ComponentAvailable StatusConditionType = "Available"
```